mods/battle-royale/hook/loc/RU/strings_db.lua

```diff
@@ -0,0 +1,100 @@
+-- delay keys
+delay_label="Королевская битва: время на подготовку"
```

Comment:
I think it is good practice to add the mod name to these values - to prevent (accidental) identical keys. As an example: `delay_label` -> `battle_royale_delay_label`.

Reply:
More often, the name of the mod in the key is added if, for example, there are similar keys in another mod. In our case, this does not make sense, it makes the key unnecessarily long and harder to read.